### DISCUSSION TITLE
Add SEO Framework by Sybre

### DISCUSSION THREAD

**Discussion by 10corp** [wpengine/faustjs#1676] [Ideas]
https://wordpress.org/plugins/autodescription/
The SEO Framework is cleaner and faster
The SEO Framework was created by a guy called Sybre Waaijer and currently has a 4.9-star rating in the WordPress Plugin directory with 90,000+ active installs.

**Comment by theodesp** [comment]
Hey @10corp thank you for the suggestion. Currently we do not have capacity or roadmap goals to extend the FaustWP plugin package to accommodate any other functionality that is not part of WordPress core. 

We also need technical knowhow of this plugin and how it works which we do not possess.

In that case someone would create another plugin that works on top of FaustWP or WP-GraphQL that handles SEO functionality the suggested plugin. 

Something similar to https://github.com/ashhitch/wp-graphql-yoast-seo  or https://github.com/AxeWP/wp-graphql-gravity-forms for example.

Therefore one suggestion for this would be to create a WPGraphQL plugin for autodescription.

I will move this ticket to discussion for further updates.

**Comment by monahmad** [comment]
Check for Compatibility:
Ensure that the SEO Framework by Sybre is compatible with the version of NamelessMC you are using. Check the documentation or release notes for both the SEO Framework and NamelessMC to verify compatibility.

Download the Plugin:
Obtain the SEO Framework plugin files. This may involve downloading a ZIP file containing the necessary files for the plugin.

Installation:
Follow the installation instructions provided by the SEO Framework plugin. This usually involves uploading the plugin files to the appropriate directory on your server.

**Comment by sybrew** [comment]
Hello! Developer of TSF here. I'm available to lend a hand; it should all be relatively straightforward.
